lib/rack/jwt/auth.rb

```diff
@@ -182,6 +187,31 @@ def missing_auth_header?(env)
         env['HTTP_AUTHORIZATION'].nil? || env['HTTP_AUTHORIZATION'].strip.empty?
       end
 
+      def auth_cookie_enabled?
+        !cookie_name.nil?
```

Comment:
```suggestion
        !auth_cookie_disabled?
```

or change `auth_cookie_disabled? method`

Reply:
Changed it the other way around so that the `!` is in the `auth_cookie_disabled?` method. Seemed to read better.